### ISSUE TITLE
Project Presentation - Dec 2

### ISSUE THREAD

**Issue by kristalow** [maryi/CSCW-Project#39]
I started google slides for our project presentation. 
Here is the link to edit the slides:
https://docs.google.com/presentation/d/150qeZcLTuqcTAVmK7QRPOW4V1q4SQNzmEZKnkOSblS8/edit?usp=sharing

**Comment by dustinfaulkner91** [comment]
Great, when would we like to get together to practice/ tidy things up?

**Comment by mpaskevi** [comment]
I suggest  a group meeting on Monday at 4:30?  We can have it in hangouts.  Please let us know if you can attend.

**Comment by mpaskevi** [comment]
I have added a few slides to the start of the presentation.  I will set the stage for the motivation to build matchmaker, based on our literature review, should not take more than a couple minutes.  

For the final presentation I think we should all speak.  Can people identify what they would like to speak about?  As stated I can start the presentation quickly introduce us and quickly explain the lit review.

**Comment by eli4** [comment]
the meeting on monday works fine for me. As for the presentation I would prefer to talk about something on the technical end since thats what im most firmiliar with.

**Comment by kristalow** [comment]
monday works for me. for the presentation i can talk about lessons we learned or challenges we faced

**Comment by maryi** [comment]
Monday is good for me too.
About the presentation I can talk after @mpaskevi about the process to define our descriptors and maybe talk again after Eli (technical stuff) to review the purpose of the survey. 
i think since @kristalow is going to work with the feedback we got from the survey, she could also mention some of that on the presentation. what do you think @kristalow ?

**Comment by kristalow** [comment]
yeah i can talk a bit about the survey results during the presentation & lessons we learned.
maybe @dustinfaulkner91 can talk about the challenges & future work since we don't have anyone assigned to those slides yet

**Comment by dustinfaulkner91** [comment]
Sure. Sounds good to me.
